### PR TITLE
Fix missing article macros and improper its/it's usage

### PR DESCRIPTION
**Pull request by liz-lavenza**
## Description of changes
Fixes many missing article macros, particularly cases like `The [src]` instead of `\The [src]`, etc.
Also fixes improper uses of its/it's.
Standardises the Marooned Ship away site character's name to Horacy Warda instead of Horazy Warda and does a pass on the grammar, spelling, and word choice for it.

## Why and what will this PR improve
I found two grammar issues and proceeded to go and fix a bunch of them. I tried to avoid doing it for datums, but some may have slipped in anyway.

### CODE REVIEW

File: code/game/machinery/biogenerator.dm [hunk 1]
```diff
@@ -86,7 +86,7 @@
 		to_chat(user, "<span class='notice'>\The [src] is currently processing.</span>")
 	if(istype(O, /obj/item/chems/glass))
 		if(beaker)
-			to_chat(user, "<span class='notice'>]The [src] is already loaded.</span>")
+			to_chat(user, "<span class='notice'>]\The [src] is already loaded.</span>")
```
Comment (MistakeNot4892) [inline code/game/machinery/biogenerator.dm:89]: Erroneous square bracket here.

File: code/game/machinery/dehumidifier.dm [hunk 2]
```diff
@@ -59,8 +59,8 @@
 			return TRUE
 		set_active(!active)
 		user.visible_message(
-			SPAN_NOTICE("[user] switches [active ? "on" : "off"] the [src]."),
-			SPAN_NOTICE("You switch [active ? "on" : "off"] the [src]."))
+			SPAN_NOTICE("[user] switches [active ? "on" : "off"] \the [src]."),
```
Comment (MistakeNot4892) [inline code/game/machinery/dehumidifier.dm:62]: `\The [user]`?

Comment (liz-lavenza) [reply code/game/machinery/dehumidifier.dm:62]: `the` to `\the` is a functional change, `[user]` to `\The [user]` is purely cosmetic.

File: code/modules/power/smes_construction.dm [hunk 3]
```diff
@@ -165,7 +165,7 @@
 			if(user_protected && prob(80))
 				to_chat(h_user, SPAN_WARNING("Small electrical arc almost burns your hand. Luckily you had your gloves on!"))
 			else
-				to_chat(h_user, SPAN_DANGER("Small electrical arc sparks and burns your hand as you touch the [src]!"))
+				to_chat(h_user, SPAN_DANGER("Small electrical arc sparks and burns your hand as you touch \the [src]!"))
```
Comment (MistakeNot4892) [inline code/modules/power/smes_construction.dm:168]: Minor note but these messages seem to be missing an article for `small electrical arc`.

**Review by MistakeNot4892 — APPROVED** [review]
Commented issues are non-blocking but would be nice to have fixed.

**Pull request merged**